src/components/change-requests/review-change-request/review-change-request.tsx

```diff
@@ -31,7 +33,12 @@ const ReviewChangeRequest: React.FC<ReviewChangeRequestProps> = ({
 
   const handleConfirm = async (e: any) => {
     e.preventDefault();
-    await mutateAsync({ crId, reviewNotes, accepted: option === 'Accept' });
+    await mutateAsync({
+      reviewerId: auth.user?.userId,
```

Comment:
This was made required, so a default value must be provided, or an assertion that it can't be undefined if that's the case. Additionally this needs to be updated through the front end hooks to actually send it to the back end.